src/elements/content-picker/itemSelectionHelper.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @flow
+ * @file Functions to determine whether a given item is selected.
+ * @author Box
+ */
+
+export function containsItem(item: BoxItem) {
```

Comment:
Can you move these exports to the bottom. It's our new convention. So

```
function foo() {}

function bar() {}

export { foo, bar };
```

Reply:
Also if you have time can you add a couple of unit tests for this file for these new functions